diff --git a/src/server/scripts/CMakeLists.txt b/src/server/scripts/CMakeLists.txt
index d44bdc3..b58bd93 100644
--- a/src/server/scripts/CMakeLists.txt
+++ b/src/server/scripts/CMakeLists.txt
@@ -46,6 +46,9 @@ if(SCRIPTS)
   include(Kalimdor/CMakeLists.txt)
   include(Outland/CMakeLists.txt)
   include(Northrend/CMakeLists.txt)
+  
+  include (${CMAKE_SOURCE_DIR}/../CataclysmScripts/CMakeLists.txt)
+  
 endif()
 
 message(STATUS "SCRIPT PREPARATION COMPLETE")
